source/fundamentals/connection/tls.txt

```diff
@@ -152,6 +155,36 @@ please refer to the `JSSE Reference Guide
 
 .. _tls-disable-hostname-verification:
 
+Configure a Client-Specific Trust Store and Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a client-specific trust store and key store using the
+``init()`` method of the ``SSLContext`` class.
```

Comment:
Question: The old documentation also mentions that you also "set the JVM default ones", should that message be conveyed here as well?

Reply:
I think it is an either or situation. You either specify the JVM defaults or a client specific one. 

> One may either configure trust stores and key stores specific to the client via javax.net.ssl.SSLContext.init(KeyManager[] km, TrustManager[] tm, SecureRandom random), or set the JVM default ones.